tests/unit/lms/services/course_test.py

```diff
@@ -112,6 +112,12 @@ def test_upsert_creates_new(self, svc, db_session):
         assert db_session.query(Course).count() == 1
         assert course.authority_provided_id == "new authority_provided_id"
 
+    def test_generate_authority_provided_id(self, svc):
```

Comment:
You could argue this is not a real unit test and I'm testing hashed_id instead of `generate_authority_provided_id`... and that's probably true but I think the alternative  wouldn't really test much I could change it to use

```
 @pytest.fixture
    def hashed_id(self, patch):
        return patch("lms.resources.lti_launch.hashed_id")
```

if we prefer that